lib/cdo/aws/cloud_formation.rb

```diff
@@ -138,32 +143,53 @@ def update_bootstrap_script
         )
       end
 
+      # Prints the latest output from a CloudWatch Logs log stream, if present.
+      def tail_log(quiet: false)
+        log_config = {
+          log_group_name: STACK_NAME,
+          log_stream_name: LOG_NAME
+        }
+        log_config[:next_token] = @@log_token unless @@log_token.nil?
+        # Return silently if log doesn't exist.
+        resp = logs.get_log_events(log_config) rescue return
```

Comment:
There might be a specific kind of exception for when the log doesn't exist, but `rescue return` is more concise and best describes the general intent (return silently if we can't get the log events _for any reason_). I'll update the comment to match.